packages/opentelemetry-instrumentation/src/platform/node/instrumentation.ts

```diff
@@ -102,6 +102,30 @@ export abstract class InstrumentationBase<T = any>
     return exports;
   }
 
+  protected getModuleVersion(directory: string): string {
+    const modulePackage = this.findModulePackage(directory);
+    if (!modulePackage) {
+      return '0.0.0';
```

Comment:
i don't think we should return a invalid version if the version isn't found, i believe `module.moduleVersion` can already be undefined

Reply:
`version` is used further down with `isSupported` and ultimately `semver.satisfies()`. I'm not familiar enough to determine the behaviour if we were to let undefined into these methods.